### PR TITLE
README.md: how to download a single example

### PR DESCRIPTION
**Pull request by noamraph**
I wanted to try an example, and I didn't manage to clone the repository or download the zip, because the repository is very big and the connection hang up. So I discovered I can use svn to download a single example. Perhaps it can be useful for others too.

### CODE REVIEW

**Comment by noamraph** [comment]
@googlebot I signed it!

**Review by RedBrogdon — COMMENTED** [review]
That's a useful tip!

Would you mind going one step further and including a line about how to use a shallow clone to get the current state of the whole repo without its entire commit history? That could also help developers on unreliable connections.

https://www.perforce.com/blog/vcs/git-beyond-basics-using-shallow-clones

**Comment by RedBrogdon** [comment]
Closing this for now, since it hasn't been updated for a month. @noamraph, if you're still interested in this PR, feel free to reopen it.

**Comment by noamraph** [comment]
@RedBrogdon I added a tip about shallow clone, as you suggested. It seems this also re-opened the PR.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,20 @@ A collection of open source samples that illustrate best practices for
 The easiest way to browse through the samples in this repo (as well as a few others!)
 is the [visual samples index](https://flutter.github.io/samples).
 
+## Tip: download a single example using svn
```
Comment (domesticmouse) [inline README.md:13]: Now that this tip isn't just about using `svn`, can we re-word the tip summary to reflect this? e.g. Tip: reducing the size of your checkout

Thoughts?

Comment (RedBrogdon) [reply README.md:13]: That's a good point. 

Something like "Tip: minimize download size" would cover both.

@noamraph, what do you think?

**Review by RedBrogdon — APPROVED** [review]

**Comment by RedBrogdon** [comment]
@noamraph, to spare you from having to come back and edit this again, I went ahead and tweaked the title, and then merged it.

Thanks for the PR!

**Comment by noamraph** [comment]
@RedBrogdon great, thanks!

**Pull request merged**